projects/storefrontapp-e2e-cypress/cypress/helpers/applied-promotions.ts

```diff
@@ -48,7 +48,8 @@ export function selectShippingAddress() {
     .should('not.be.empty');
   cy.get('.cx-card-title').should('contain', 'Default Shipping Address');
   cy.get('.card-header').should('contain', 'Selected');
-  cy.visit(`/checkout/delivery-mode`);
+  cy.get('button.btn-primary').click();
+  // TODO: make it more stable when JaloError happens
```

Comment:
what about this TODO?

Reply:
Left for the future, as I don't have yet a fix for that. That's also the reason why I kept this test as a "flaky".